os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -1185,6 +1185,24 @@ int rtw_printf_info(const char *format,...)
 	return ret;
 }
 
+/* Blocks the currently running task, need to called within the currently running task.
+ * It is added because there is no task suspension related function found in the SDK.
+ */
+void rtw_block_current_task(void){
+       FAR struct tcb_s *tcb;
+       tcb = this_task();
+       up_block_task(tcb, TSTATE_WAIT_FIN);
+}
+
+/* Unblocks the task previously blocked by rtw_block_current_task(), the input *task
+ * is the pid of the previously blocked task.
+ */
+void rtw_unblock_task(void *task){
+       FAR struct tcb_s *tcb;
+       tcb = sched_gettcb(*(pid_t *)task);
+       up_unblock_task(tcb);
```

Comment:
This is the schedule API which should be controlled by TizenRT scheduler. Please let me know your usage of this. Without this, it's difficult to allow to use from here.

Reply:
Dear @sunghan-chang ,
For EAP feature we need to use rtw_suspend_task() and rtw_resume_task()(previously we call freertos API to realize these two functions), but after implementation we found we cannot realize full functionality of rtw_suspend_task() and rtw_resume_task() using tizenrt api, so we rename it as rtw_block_current_task() and rtw_unblock_task(). Do you think this is ok?
Thank you.